### PR TITLE
WebsiteDocs-1: Translated the Contribution page to English

### PR DESCRIPTION
**Pull request by IamMujuziMoses**
WebsiteDocs-1: Translated the Contribution page to English

### CODE REVIEW

**Comment by IamMujuziMoses** [comment]
@dailidong @lgcareer Review this please, Thnx

**Comment by IamMujuziMoses** [comment]
This PR is associated to this main [issue]

[issue]: https://github.com/apache/incubator-dolphinscheduler/issues/3562

**Comment by CalvinKirs** [comment]
@Segun-Ogundipe @Tianqi-Dotes PTAL.Thx

**Comment by lgcareer** [comment]
@IamMujuziMoses Thx,I will review it lator.

File: docs/en-us/development/document.md [hunk 1]
```diff
@@ -1 +1,67 @@
...
+
+4. Run `npm install docsite-g`
+
+Now you can run and build a website in your local environment.
+
+### The document specification.
+
+1. Between Chinese characters and English numbers ** Spaces are Required ** Between Chinese punctuation mark and English numbers ** Spaces are not required ** to enhance the aesthetics and readability of the Chinese-English mix.
+
+2. It is recommended that you use "you" in general. Of course, you can use the term "you" when necessary, such as when there is a warning prompt.
```
Comment (Tianqi-Dotes) [inline docs/en-us/development/document.md:52]: the second  "you" maybe change to a more polite word

File: docs/en-us/development/document.md [hunk 2]
```diff
@@ -1 +1,67 @@
...
+
+3. Run “nvm use v8.16.0” to switch the current working environment to node v8.
+
+4. Run `npm install docsite-g`
+
+Now you can run and build a website in your local environment.
+
+### The document specification.
+
+1. Between Chinese characters and English numbers ** Spaces are Required ** Between Chinese punctuation mark and English numbers ** Spaces are not required ** to enhance the aesthetics and readability of the Chinese-English mix.
```
Comment (Tianqi-Dotes) [inline docs/en-us/development/document.md:50]: from the original doc, English and numbers are separated.
** Spaces are Required ** between Chinese characters and English or numbers and  ** Spaces are not required ** between Chinese punctuation marks and English or numbers, to enhance the aesthetics and readability of the Chinese-English mix.

**Review by Tianqi-Dotes — CHANGES_REQUESTED** [review]
hi MujuziMoses,
I have reviewed both docs, please check the suggestions and try to fix problems.
Thanx
Sincerely
Tianqi

**Comment by IamMujuziMoses** [comment]
Thanx @Tianqi-Dotes, really appreciate the time for review, let me get those fixed right away

File: docs/en-us/development/contribute.md [hunk 3]
```diff
@@ -1 +1,40 @@
-# todo
\ No newline at end of file
+## Contribute
+
+First of all, thank you very much for choosing and using Dolphin Scheduler, and welcome to join the Dolphin Scheduler family!
```
Comment (lgcareer) [inline docs/en-us/development/contribute.md:3]: It's better using DolphinScheduler not Dolphin Scheduler

**Review by khadgarmage — APPROVED** [review]
+1

**Review by lgcareer — APPROVED** [review]
+1

**Pull request merged**